[2024-04-29T04:21:51.109+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-04-29T04:21:51.175+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl.upload_file_geolocation scheduled__2024-04-28T00:00:00+00:00 [queued]>
[2024-04-29T04:21:51.234+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl.upload_file_geolocation scheduled__2024-04-28T00:00:00+00:00 [queued]>
[2024-04-29T04:21:51.247+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2024-04-29T04:21:51.338+0000] {taskinstance.py:2327} INFO - Executing <Task(LocalFilesystemToGCSOperator): upload_file_geolocation> on 2024-04-28 00:00:00+00:00
[2024-04-29T04:21:51.434+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'etl', 'upload_file_geolocation', 'scheduled__2024-04-28T00:00:00+00:00', '--job-id', '451', '--raw', '--subdir', 'DAGS_FOLDER/etl.py', '--cfg-path', '/tmp/tmptrz1vyrv']
[2024-04-29T04:21:51.448+0000] {standard_task_runner.py:91} INFO - Job 451: Subtask upload_file_geolocation
[2024-04-29T04:21:51.460+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=251) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-04-29T04:21:51.473+0000] {standard_task_runner.py:63} INFO - Started process 389 to run task
[2024-04-29T04:21:51.682+0000] {task_command.py:426} INFO - Running <TaskInstance: etl.upload_file_geolocation scheduled__2024-04-28T00:00:00+00:00 [running]> on host b619dce40a27
[2024-04-29T04:21:52.338+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl' AIRFLOW_CTX_TASK_ID='upload_file_geolocation' AIRFLOW_CTX_EXECUTION_DATE='2024-04-28T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-04-28T00:00:00+00:00'
[2024-04-29T04:21:52.339+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-04-29T04:21:52.405+0000] {connection.py:274} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2024-04-29T04:21:52.487+0000] {base.py:84} INFO - Using connection ID 'my_gcp_conn' for task execution.
[2024-04-29T04:21:55.856+0000] {gcs.py:557} INFO - File /opt/***/dags/data/olist_geolocation_dataset.csv uploaded to olist_geolocation_dataset.csv in swu-ds-525-112233 bucket
[2024-04-29T04:21:55.857+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-04-29T04:21:55.867+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=etl, task_id=upload_file_geolocation, execution_date=20240428T000000, start_date=20240429T042151, end_date=20240429T042155
[2024-04-29T04:21:55.901+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-04-29T04:21:55.923+0000] {taskinstance.py:3482} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-04-29T04:21:55.929+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
